app/views/projects/new.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="container feature">
 <h1>New Project</h1>
 
-<%= render 'form', project: @project %>
+<%= render 'form', project: @project, cached: true %>
```

Comment:
Caching forms won't make any significant improvement, also it can cause weird bugs

Reply:
Ok, I will see what other areas can be cached